source/common/upstream/outlier_detection_impl.h

```diff
@@ -148,6 +201,8 @@ class DetectorImpl : public Detector, public std::enable_shared_from_this<Detect
   std::list<ChangeStateCb> callbacks_;
   std::unordered_map<HostPtr, DetectorHostSinkImpl*> host_sinks_;
   EventLoggerPtr event_logger_;
+  // Factor to multiply the stdev of a cluster's Success Rate for success rate outlier ejection.
+  static constexpr double sr_stdev_factor_ = 1.9;
```

Comment:
Use our way of defining it  ex https://github.com/lyft/envoy/blob/master/source/common/router/router_ratelimit.cc#L9